index.html

```diff
@@ -150,17 +150,19 @@ <h2>A comprehensive framework for accurate and efficient RNA-Seq analysis</h2>
             
         </div>
         
-    <h2>Publication [Open access]</h2>
+    <h2>Publication</h2>
```

Comment:
This file should also be updated on the gh-pages branch since that it used for the github.io.

Reply:
I checked and the github.io page (https://bioinform.github.io/rnacocktail/) is already up to date.